Gcpe.Hub.API/appsettings.json

```diff
@@ -1,6 +1,7 @@
 {
   "Jwt": {
     "Authority": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/zdw7o87p",
+    "AuthorizationUrl": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/zdw7o87p/protocol/openid-connect/auth",
```

Comment:
Should leave this blank? Not sure we'd want the url of our keycloak server being public although it won't matter unless you have an IDIR. Up for debate really :/

Reply:
hmm good point. We will have to put this in environment settings anyways so that might be a good time to address this. With the Authority url that's already in there someone could find this authorizationurl easily enough

Reply:
True, makes sense

Reply:
To be removed?

Reply:
removed now. We will all need to add the Jwt settings to our secrets